travis-init.sh

```diff
@@ -39,4 +39,18 @@ if [[ "$TRAVIS_PHP_VERSION" != "hhvm" &&
         echo "extension=libev.so" >> "$(php -r 'echo php_ini_loaded_file();')"
     fi
 
+    # install 'libuv' PHP extension (does not support php 5)
+    if [[ "$TRAVIS_PHP_VERSION" = "7.0" ||
+          "$TRAVIS_PHP_VERSION" = "7.1" ||
+          "$TRAVIS_PHP_VERSION" = "7.2" ]]; then
+        git clone --recursive https://github.com/bwoebi/php-uv
+        pushd php-uv
+        phpize
+        ./configure
+        make
+        make install
+        popd
+        echo "extension=uv.so" >> "$(php -r 'echo php_ini_loaded_file();')"
+    fi
```

Comment:
Should this install from PECL instead (see above)?

Reply:
Installing through PECL now :tada: